Source/WTF/wtf/FixedVector.h

```diff
@@ -196,8 +194,8 @@ class FixedVector {
 
     Storage* storage() { return m_storage.get(); }
 
-    std::span<const T> span() const { return { m_storage ? m_storage->data() : nullptr, size() }; }
-    std::span<T> mutableSpan() { return { m_storage ? m_storage->data() : nullptr, size() }; }
+    std::span<const T> span() const { return unsafeMakeSpan(m_storage ? m_storage->data() : nullptr, size()); }
+    std::span<T> mutableSpan() { return unsafeMakeSpan(m_storage ? m_storage->data() : nullptr, size()); }
```

Comment:
TrailingArray offers a span() accessor, so you could do m_storage->span().subspan(0, size()), or something like that.

Reply:
Good point. I'm following up in https://github.com/WebKit/WebKit/pull/36450.